docs/_data/icons.json5

```diff
@@ -1,67 +1,228 @@
 [
-  'account',
-  'angle-bottom',
-  'angle-down',
-  'angle-end',
-  'angle-left',
-  'angle-right',
-  'angle-start',
-  'angle-top',
-  'angle-up',
-  'bell',
-  'billing',
-  'calendar',
-  'checkmark',
-  'checkmark-circle',
-  'cog',
-  'copy',
-  'download',
-  'envelope',
-  'exclamation-circle',
-  'exclamation-diamond',
-  'exclamation-triangle',
-  'export',
-  'external-link',
-  'filter',
-  'help-circle',
-  'info-circle',
-  'input-file',
-  'input-time',
-  'input-url',
-  'kbd-arrow-down',
-  'kbd-arrow-left',
-  'kbd-arrow-right',
-  'kbd-arrow-up',
-  'kbd-capslock',
-  'kbd-command',
-  'kbd-delete',
-  'kbd-eject',
-  'kbd-option',
-  'kbd-return',
-  'kbd-shift',
-  'kbd-space',
-  'kbd-tab',
-  'lock',
-  'minus',
-  'minus-circle',
-  'monitoring',
-  'payment',
-  'pencil',
-  'phone',
-  'plus',
-  'plus-or-minus',
-  'search',
-  'server',
-  'sort',
-  'sort-down',
-  'sort-up',
-  'support',
-  'tag',
-  'technical-change',
-  'technical-incident',
-  'ticketing',
-  'times',
-  'times-circle',
-  'trash',
-  'user',
+    {
+        name: 'account',
+        minver: '0.1.10',
+    },
```

Comment:
Given that we'll need extra metadata about icons in future work, I've updated the data file for the icons, in preparation.